src/main/java/org/commcare/cases/entity/NodeEntityFactory.java

```diff
@@ -73,10 +73,7 @@ public Detail getDetail() {
                  * We are wrapping the original exception in a new XPathException to avoid
                  * refactoring large number of functions caused by throwing XPathSyntaxException here.
                  */
-                XPathException xe = new XPathException(e.getMessage(), e);

```

Comment:
What will be the value of `xe.getMessage()` now that you are not explicitly passing a message to the constructor? We would want `xe.getMEssage()` to be equivalent to `e.getMessage()`; can you confirm that that is the case?